website/content/en/preview/tasks/provisioning.md

```diff
@@ -113,7 +113,40 @@ spec:
 ```
 In order for a pod to run on a node defined in this provisioner, it must tolerate `nvidia.com/gpu` in its pod spec.
 
-### Example: Adding the Cilium Startup Taint
+## Example: Creating Provisioner Hierarchies with Weights
```

Comment:
I think this is better framed as Preferences to map to existing concepts of pod level preferences. This should include some discussion about whether or not the instance types are available. 

Thinking more on this feature -- how does this relate to @tzneal's consolidation work? Will consolidations be considered across weights?